Game/Assets/Scenes/Production/MainMenu.scene

```diff
@@ -641,11 +641,7 @@
                     "Character_selector1": 3292514778,
                     "Name9": "Character_selector2",
                     "Character_selector2": 1972725956,
-                    "Name10": "confirm_multi",
```

Comment:
This is a placeholder for Irene text that say "Press space for multi " or "Press space for single".  Don't remove it

Reply:
Doing tests in this specific menu we found out that it was super confusing to use. So I discussed with Irene, and Im changing it assets/placements in order to make it more usable-